storagenode/reputation/reputation.go

```diff
@@ -16,6 +16,8 @@ type DB interface {
 	Store(ctx context.Context, stats Stats) error
 	// Get retrieves stats for specific satellite
 	Get(ctx context.Context, satelliteID storj.NodeID) (*Stats, error)
+	// All retrieves all stats from DB
+	All(ctx context.Context) ([]Stats, error)
```

Comment:
This sounds like a bad idea to query the full database.

Reply:
@egonelbre num of rows for this query should be equal to `len(trustpool)`, so instead of querying each `reputation` in `for loop range trustpool` I created this query.